data/janet-dev.json

```
@@ -3,7 +3,7 @@
   "url": "https://janet-dev.github.io",
   "image": "https://github.com/janet-dev.png",
   "why": "To be not so lazy",
-  "weekly": ["cycle to work every day"],
+  "weekly": ["For 5 days, do over 10k steps daily, drink more water"],
```

```suggestion
  "weekly": ["For 5 days", "do over 10k steps daily", "drink more water"],
```

I guess you want these 3 items to be seperate items, instead of a single string